CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Changelog
 
+## [1.56](https://github.com/brave/brave-ios/releases/tag/v1.56)
+
+### Web3
+
+- Implemented “Select a token to send” view which allows selecting an account and an asset at the same time. ([#7523](https://github.com/brave/brave-ios/issues/7523))
+- Removed network name from the “Public Encryption Key Request” panel. ([#7495](https://github.com/brave/brave-ios/issues/7495))
+- Improved NFT importing in Brave Wallet by introducing a new button to add custom assets quicker in NFT tab view and also simplifying the “Add Custom Asset” form by moving moving “Token address” field to the top. ([#7556](https://github.com/brave/brave-ios/issues/7556))
+- Updated “Select a Token to Send” screen to sort test network tokens to be displayed at the end. ([#7640](https://github.com/brave/brave-ios/issues/7640))
+- Updated network filters to allow multiple networks to be selected. ([#7607](https://github.com/brave/brave-ios/issues/7607))
+- Updated the “Secondary Networks” list in the “Select Network Filter” screen by moving “Filecoin EVM Testnet” to be included with the other testnets. ([#7583](https://github.com/brave/brave-ios/issues/7583))
+- Resolved migrating user assets when restoring from the wallet lock screen. ([#7785](https://github.com/brave/brave-ios/issues/7785))
+- Resolved asset preferences from being overwritten when auto-discovery returns results. ([#7443](https://github.com/brave/brave-ios/issues/7443))
+- Fixed auto-discovered assets from being duplicated after restoring Brave Wallet. ([#7804](https://github.com/brave/brave-ios/issues/7804))
+- Fixed “Send” screen in Wallet to retain the last selected token when opened or display the native token by default. ([#7759](https://github.com/brave/brave-ios/issues/7759))
```

Comment:
Change to `Fixed “Send” screen in Wallet from retaining the last selected token when opened`.